src/tasks/WasmAppBuilder/PInvokeTableGenerator.cs

```diff
@@ -443,15 +443,15 @@ private void EmitNativeToInterp(StreamWriter w, ref List<PInvokeCallback> callba
             pindex = 0;
             if (!is_void)
             {
-                sb.Append("&res");
+                sb.Append("(int*)&res");
```

Comment:
This change introduced `UnmanagedCallersOnly` callback with long argument. It caused wasm build to fail with `pinvoke-table.h(303,87): error G55D6342F: (NETCORE_ENGINEERING_TELEMETRY=Build) incompatible pointer types passing 'int64_t *' (aka 'long long *') to parameter of type 'int *'`. @radical Is this the right fix for this build error?

Reply:
It looks fine.